core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -2268,6 +2268,9 @@ public static PropertyKey fromString(String input) {
   /** Property name. */
   private final String mName;
 
+  /** Property Key description. */
+  private String mDescription;
```

Comment:
final may not be a good choice since mDescription will be changed in method create() call for each Property Key.